docs/content/intro/writing-efficient-pipelines.md

```diff
@@ -20,6 +20,3 @@ Here are a few things to help you:
 - DO NOT try to load external JavaScript modules (like Node modules). This is not supported.
 
 If you've gone through this article, then you're probably scratching your head on what to [read next][readnext].
-
-
-[readnext]: readnext.md
```

Comment:
This line will need to stay if it is still being referenced in-line (line 22 above).